CONTRIBUTING.md

```diff
@@ -12,10 +12,6 @@ Thanks for being willing to contribute!
 3. `$ yarn test` to validate you've got it working
 4. Create a branch for your PR
 
-## Here as part of Hacktoberfest?
-
-Head over to [here](https://hacktoberfest.digitalocean.com/sign_up/register) to signup if you haven't already
```

Comment:
Are you no longer interested in participating in Hacktoberfest?

Reply:
Happy to participate but it's once a year thing so probably doesn't need to be in the contributing file all year round (instead we can just tag issues with the `hacktoberfest` label)